scripts/api/data/licenses/licenseMIT.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "MIT License",
+  "uri": "https://spdx.org/licenses/MIT.html",
+  "shortDescription": "MIT License (MIT).",
```

Comment:
I'm just wondering where this short description comes from.

Reply:
I am open to suggestions 🤣 

` Massachusetts Institute of Technology License (MIT).`?